packages/components/src/toolbar/toolbar-dropdown-menu/index.tsx

```diff
@@ -37,7 +37,10 @@ function ToolbarDropdownMenu(
 					popoverProps={ {
 						...props.popoverProps,
 					} }
-					toggleProps={ toolbarItemProps }
+					toggleProps={ {
+						size: 'compact',
+						...toolbarItemProps,
+					} }
```

Comment:
As seen in the default Storybook story:

<img width="554" alt="ToolbarDropdownMenu" src="https://github.com/user-attachments/assets/cf3e072e-ea1c-48d1-95e0-6773cbcbd91c" />